### PR TITLE
polls-2024-02-26

### PR DESCRIPTION
**Pull request by VoteWizard**

### CODE REVIEW

File: governance/polls/Smart Burn Engine - Rate of MKR Accumulation Change - February 26, 2024.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Smart Burn Engine - Rate of MKR Accumulation Change - February 26, 2024
+summary: Signal your support or opposition to the changing the Smart Burn Engine Rate of MKR Accumulation
```
Comment (MakerJanSky) [inline governance/polls/Smart Burn Engine - Rate of MKR Accumulation Change - February 26, 2024.md:3]:
```suggestion
summary: Signal your support or opposition to changing the Smart Burn Engine Rate of MKR Accumulation.
```

File: governance/polls/Smart Burn Engine - Rate of MKR Accumulation Change - February 26, 2024.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following Smart Burn Engine (SBE) change:
+
+- Increase The [Rate Of MKR Accumulation](https://mips.makerdao.com/mips/details/MIP104#9-1-2a) by 20M DAI per year from 60M DAI per year to **80M DAI per year**.
+
+To implement this change, the following Smart Burn Engine (SBE) parameter will be adjusted:
+
+- Decrease the [`hop` parameter](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1a) for 6,570 seconds from 26,280 seconds to **19,710 seconds**.
```
Comment (MakerJanSky) [inline governance/polls/Smart Burn Engine - Rate of MKR Accumulation Change - February 26, 2024.md:35]:
```suggestion
- Decrease the [`hop` parameter](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1a) 6,570 seconds from 26,280 seconds to **19,710 seconds**.
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
+---
+title: SparkLend Ethereum - Activate Cap Automator - February 26, 2024
+summary: Signal your support or opposition to activating the SparkLend Cap Automator
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:3]:
```suggestion
summary: Signal your support or opposition to activating the SparkLend Cap Automator.
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to activating the SparkLend Cap Automator. This new feature will enable dynamic adjustments to borrow and supply caps. The core mechanism is similar to the [Debt Ceiling Instant Access Modules (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam) used for Maker core vaults and will help reduce governance overhead in maintaining the supply and borrow caps for various assets.
+
+If approved, the module will be activated with the following max limit (max), buffer size (gap) and cooldown (ttl) parameters for the covered assets:
+
+#### DAI
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:33]: I suggest removing this subsection as no changes to the DAI D3M are needed.

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+* Supply cap
+  * gap: 150,000 WETH
+  * ttl: 12 hours
+  * max: 2 million WETH
+* Borrow cap
+  * gap: 10,000 WETH
+  * ttl: 12 hours
+  * max: 1 million WETH
+
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:47]:
```suggestion
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+* Supply cap
+  * gap: 500 WBTC
+  * ttl: 12 hours
+  * max: 5,000 WBTC
+* Borrow cap
+  * gap: 100 WBTC
+  * ttl: 12 hours
+  * max: 2,000 WBTC
+
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:81]:
```suggestion
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+#### sDAI
+
+* Supply cap
+  * gap: 50 million sDAI
+  * ttl: 12 hours
+  * max: 1 billion sDAI
+* Borrow cap: n/a - not a borrowable asset
+
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:90]:
```suggestion
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+* Supply cap
+  * gap: n/a
+  * ttl: n/a
+  * max: 30 million USDT
+* Borrow cap
+  * gap: 3 million USDT
+  * ttl: 12 hours
+  * max: 28.5 million USDT
+
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:113]:
```suggestion
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to activating the SparkLend Cap Automator. This new feature will enable dynamic adjustments to borrow and supply caps. The core mechanism is similar to the [Debt Ceiling Instant Access Modules (DC-IAM)](https://manual.makerdao.com/module-index/module-dciam) used for Maker core vaults and will help reduce governance overhead in maintaining the supply and borrow caps for various assets.
+
+If approved, the module will be activated with the following max limit (max), buffer size (gap) and cooldown (ttl) parameters for the covered assets:
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:31]:
```suggestion
If approved, the module will be activated with the following max limit (max), buffer size (gap), and cooldown (ttl) parameters for the covered assets:
```

File: governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
+---
+title: SparkLend Ethereum - Collateral Parameter Changes - February 26, 2024
+summary: Signal your support or opposition to the SparkLend Collateral Parameter Changes
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md:3]:
```suggestion
summary: Signal your support or opposition to the SparkLend Collateral Parameter Changes.
```

File: governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+* Increase LTV from 74% to **79%**
+* Increase LT from 76% to **80%**
+* Increase liquidation penalty from 4.5% to **5%**
+
+#### ETH emode
+
+* Increase LTV from 90% to **92%**
+
+
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md:62]:
```suggestion
```

File: governance/polls/Smart Burn Engine - Rate of MKR Accumulation Change - February 26, 2024.md (hunk 1 above)
Comment (blimpa) [inline governance/polls/Smart Burn Engine - Rate of MKR Accumulation Change - February 26, 2024.md:3]: Either "to changing" or "to the changing of"

Also, full stop is missing

File: governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+# Poll: SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, February 26 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md:27]: We still lack comprehensive documentation for Spark Parameters. I recommend either linking to AAVE's documentation or providing a brief explanation of the parameters being modified.

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
Nice work, VoteWizard.

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 80,000 rETH
+* Borrow cap
+  * gap: 100 rETH
+  * ttl: 12 hours
+  * max: 2,400 rETH
+
+#### WBTC
+
+* Supply cap
+  * gap: 500 WBTC
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:74]:
```suggestion
  * `gap`: 500 WBTC
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+  * ttl: 12 hours
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:75]:
```suggestion
  * `ttl`: 12 hours
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 5,000 WBTC
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:76]:
```suggestion
  * `max`: 5,000 WBTC
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+* Borrow cap
+  * gap: 100 WBTC
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:78]:
```suggestion
  * `gap`: 100 WBTC
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+  * ttl: 12 hours
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:79]:
```suggestion
  * `ttl`: 12 hours
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 2,000 WBTC
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:80]:
```suggestion
  * `max`: 2,000 WBTC
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+
+#### sDAI
+
+* Supply cap
+  * gap: 50 million sDAI
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:86]:
```suggestion
  * `gap`: 50 million sDAI
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 20]
```diff
@@ -0,0 +1,137 @@
...
+  * ttl: 12 hours
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:87]:
```suggestion
  * `ttl`: 12 hours
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 21]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 1 billion sDAI
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:88]:
```suggestion
  * `max`: 1 billion sDAI
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 22]
```diff
@@ -0,0 +1,137 @@
...
+* Borrow cap: n/a - not a borrowable asset
+
+
+#### USDC
+
+* Supply cap
+  * gap: n/a
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:95]:
```suggestion
  * `gap`: n/a
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 23]
```diff
@@ -0,0 +1,137 @@
...
+  * ttl: n/a
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:96]:
```suggestion
  * `ttl`: n/a
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 24]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 60 million USDC
+* Borrow cap
+  * gap: 6 million USDC
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:99]:
```suggestion
  * `gap`: 6 million USDC
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 25]
```diff
@@ -0,0 +1,137 @@
...
+  * ttl: 12 hours
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:100]:
```suggestion
  * `ttl`: 12 hours
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 26]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 57 million USDC
+
+#### USDT
+
+* Supply cap
+  * gap: n/a
+  * ttl: n/a
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:107]:
```suggestion
  * `ttl`: n/a
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 27]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 30 million USDT
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:108]:
```suggestion
  * `max`: 30 million USDT
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 28]
```diff
@@ -0,0 +1,137 @@
...
+* Borrow cap
+  * gap: 3 million USDT
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:110]:
```suggestion
  * `gap`: 3 million USDT
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 29]
```diff
@@ -0,0 +1,137 @@
...
+  * ttl: 12 hours
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:111]:
```suggestion
  * `ttl`: 12 hours
```

File: governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md [hunk 30]
```diff
@@ -0,0 +1,137 @@
...
+  * max: 28.5 million USDT
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Activate Cap Automator - February 26, 2024.md:112]:
```suggestion
  * `max`: 28.5 million USDT
```

File: governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md (hunk 10 above)
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md:3]: Add full stop

File: governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md [hunk 31]
```diff
@@ -0,0 +1,83 @@
...
+#### sDAI
+
+* Increase LTV from 74% to **79%**
+* Increase LT from 76% to **80%**
+* Increase liquidation penalty from 4.5% to **5%**
+
+#### ETH emode
+
+* Increase LTV from 90% to **92%**
+
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md:61]: Remove extra line

File: governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md [hunk 32]
```diff
@@ -0,0 +1,83 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following [SparkLend Ethereum](https://app.spark.fi/markets/?marketName=proto_spark_v3) Collateral Parameter Changes.
+
+#### WETH
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md:31]: [This table](https://makerdao-forum-backup.s3.dualstack.us-east-1.amazonaws.com/original/3X/6/f/6fe6d0181f8494eee4aa9f6ace857ba91c6b73b0.png) shows other parameters have changed, too. Not sure why those were left out later in the original post

File: governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md [hunk 33]
```diff
@@ -0,0 +1,83 @@
...
+
+* Increase LTV from 80% to **82%**
+* Increase LT from 82.5% to **83%**
+* Reduce slope 1 from 3.2% to **2.8%**
```
Comment (blimpa) [inline governance/polls/SparkLend Ethereum - Collateral Parameter Adjustments - February 26, 2024.md:35]: Maybe "slope 1 (optimal borrowing rate)"?

**Review by blimpa — CHANGES_REQUESTED** [review]

**Pull request merged**